lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,9 @@
+class RakeArguments
+  def self.task_options
+    ARGV.drop_while { |e| e != '--' }.drop(1).join(' ')
+  end
+
+  def self.more_tasks?
+    ARGV.take_while { |e| e != '--' }.drop(1).any?
```

Comment:
I agree with @Insti on lines 3 and 7.  Let the block be the end of it, let the return be the return.  This does suggest a couple of problems with the encapsulation on its own, though.  Let it be a class, instantiate it directly, and have it return itself, and respond to `any?` and `to_s`